src/crawlee/storage_clients/_file_system/_request_queue_client.py

```diff
@@ -115,9 +115,8 @@ def __init__(
 
         self._state = RecoverableState[RequestQueueState](
             default_state=RequestQueueState(),
-            persist_state_key='request_queue_state',
+            persist_state_key=f'request_queue_{self._metadata.id}_state',
```

Comment:
We use these in the SDK for input and other internal thing (storage alias mapping):

<img width="200" height="412" alt="image" src="https://github.com/user-attachments/assets/e861bb4d-e5f1-46dd-b059-513faf954d0b" />

So to stay with those, please use:

```
__RQ_STATE_{self._metadata.id}
```